Frontend/ui-driver/src/Screens/DriverDetailsScreen/View.purs

```diff
@@ -249,12 +261,22 @@ driverSubsection push state title =
   , height WRAP_CONTENT
   , text (getString EDIT)
   , gravity RIGHT
-  , visibility if(title == DRIVER_ALTERNATE_MOBILE_INFO) then VISIBLE else GONE
+  , visibility case title of
+                DRIVER_ALTERNATE_MOBILE_INFO -> case state.data.driverAlternateMobile of
+                                                Just _ -> VISIBLE
+                                                _ -> GONE
+                GENDER_INFO -> case state.data.driverGender of
+                                                Just _ -> VISIBLE
```

Comment:
Please refactor this case match as it is not required, and try to use create a function for multiple cases